sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -14,13 +15,34 @@ namespace Azure.Monitor.Ingestion
 {
     /// <summary> The IngestionUsingDataCollectionRules service client. </summary>
     [CodeGenClient("IngestionUsingDataCollectionRulesClient")]
+    [CodeGenSuppress("LogsIngestionClient", typeof(Uri), typeof(TokenCredential), typeof(LogsIngestionClientOptions))]
     public partial class LogsIngestionClient
     {
         /// <summary> Initializes a new instance of LogsIngestionClient for mocking. </summary>
         protected LogsIngestionClient()
         {
         }
 
+        /// <summary> Initializes a new instance of LogsIngestionClient. </summary>
+        /// <param name="endpoint"> The Data Collection Endpoint for the Data Collection Rule, for example https://dce-name.eastus-2.ingest.monitor.azure.com. </param>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> or <paramref name="credential"/> is null. </exception>
+        public LogsIngestionClient(Uri endpoint, TokenCredential credential, LogsIngestionClientOptions options)
+        {
+            Argument.AssertNotNull(endpoint, nameof(endpoint));
+            Argument.AssertNotNull(credential, nameof(credential));
+            options ??= new LogsIngestionClientOptions();
+
+            ClientDiagnostics = new ClientDiagnostics(options, true);
+            _tokenCredential = credential;
+            var authorizationScope = $"{(string.IsNullOrEmpty(options.Audience?.ToString()) ? IngestionAudience.AzurePublicCloud : options.Audience)}";
```

Comment:
Consider using the null-coalescing operator - 
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator

Reply:
I think that operator doesn't work if the string is empty and we want to check for that as well